e2e/nx-aws-cache-e2e/dist/packages/nx-aws-cache/generators.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
what are `dist` files doing in the repo?

Reply:
Fixed it was missing **/dist in .gitignore